### PR TITLE
refactor(jsx2mp): webpack can be configured when compiling miniapp component

### PR DESCRIPTION
**Pull request by ChrisCindy**
- Remove jsx2mp-cli dependency and use built-in webpack config to compile miniapp
- Optimize output log

### CODE REVIEW

File: packages/build-plugin-rax-component/src/config/miniapp/getBase.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+  config.externals([
+    function(ctx, request, callback) {
+      if (/^@core\//.test(request)) {
+        return callback(null, `commonjs2 ${request}`);
+      }
+      if (/\.(css|sass|scss|styl|less)$/.test(request)) {
+        return callback(null, `commonjs2 ${request}`);
+      }
+      if (/^@weex-module\//.test(request)) {
```
Comment (SoloJiang) [inline packages/build-plugin-rax-component/src/config/miniapp/getBase.js:141]: 快应用要加吗

File: packages/build-plugin-rax-component/src/build.js [hunk 2]
```diff
@@ -51,12 +62,40 @@ module.exports = (api, options = {}) => {
...
+      const distDir = options[MINIAPP].distDir || `${outputDir}/${MINIAPP}`;
+      console.log('   ', chalk.underline.white(path.resolve(rootDir, distDir)));
+      console.log();
+    }
+    if (targets.includes(WECHAT_MINIPROGRAM)) {
+      console.log(chalk.green('[WeChat MiniProgram] Component lib at:'));
+      const distDir = options[WECHAT_MINIPROGRAM].distDir || `${outputDir}/${WECHAT_MINIPROGRAM}`;
+      console.log('   ', chalk.underline.white(path.resolve(rootDir, distDir)));
+      console.log();
+    }
```
Comment (SoloJiang) [inline packages/build-plugin-rax-component/src/build.js:86]: 话说小程序这块能用 platformConfig 那个适配文件用个数组遍历打印？想到后面加一个小程序就要复制粘贴一下就很难受。

File: packages/build-plugin-rax-component/src/dev.js [hunk 3]
```diff
@@ -135,28 +102,43 @@ module.exports = (api, options = {}) => {
...
+      let outputPath = '';
+      if (options[WECHAT_MINIPROGRAM].distDir) {
+        outputPath = options[WECHAT_MINIPROGRAM].distDir;
+        console.log(chalk.green('[WeChat MiniProgram] Component lib at: '));
+      } else {
+        outputPath = `demo/${WECHAT_MINIPROGRAM}`;
+        console.log(chalk.green('[WeChat MiniProgram] Use wechat miniprogram developer tools to open the following folder:'));
+      }
+      console.log('   ', chalk.underline.white(path.resolve(rootDir, outputPath)));
       console.log();
```
Comment (SoloJiang) [inline packages/build-plugin-rax-component/src/dev.js:127]: +1

File: packages/build-plugin-rax-component/src/config/miniapp/getBase.js (hunk 1 above)
Comment (ChrisCindy) [reply packages/build-plugin-rax-component/src/config/miniapp/getBase.js:141]: 快应用编译走的是 web 的链路

**Pull request merged**